### PR TITLE
Docker Setup

### PR DESCRIPTION
**Pull request by egbaydarov**
Add simple GPU-supported Docker setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,17 @@ Check out the original [paper](https://vcai.mpi-inf.mpg.de/projects/DragGAN/) fo
 
 ## Running Locally
 
+### With Docker 
+
+```bash
+git clone https://github.com/Zeqiang-Lai/DragGAN.git
+cd DragGAN
+docker build -t dragcan .
+docker run -t -p 7860:7860 --gpus all dragcan
```
Comment (Zeqiang-Lai) [inline README.md:72]: there is a typo in dragcan

File: README.md [hunk 2]
```diff
@@ -63,6 +63,17 @@ Check out the original [paper](https://vcai.mpi-inf.mpg.de/projects/DragGAN/) fo
 
 ## Running Locally
 
+### With Docker 
+
```
Comment (Zeqiang-Lai) [inline README.md:67]: Could you add a link to the docker official website ?

**Comment by Zeqiang-Lai** [comment]
Looks great. Please resolve my comments above. I will find some time to test it and then it could be merged.

**Comment by egbaydarov** [comment]
I believe this is a draft that you can enhance as needed. Okay, I'll attempt to finalize it.

File: README.md (hunk 2 above)
Comment (egbaydarov) [reply README.md:82]: done

File: README.md (hunk 1 above)
Comment (egbaydarov) [reply README.md]: done

**Comment by egbaydarov** [comment]
so, i also push image into. If u want u can push into your own simply using.
```
# after building image
docker tag draggan:tatest <your-repo>/draggan:latest
docker push <your-repo>/draggan:latest
```
 and update readme

**Pull request merged**